tests/generic.rs

```diff
@@ -77,3 +77,50 @@ mod test_convert_x509_subject_alternative_name {
 		assert!(actual.subject_alt_names.contains(&ip_san));
 	}
 }
+
+#[cfg(feature = "x509-parser")]
+mod test_x509_parser_crl {
```

Comment:
I put these tests in `generic.rs` but perhaps it would be better to create a x509-parser.rs test file?

Reply:
It's fine I think. the mod right above is doing the same.